### DISCUSSION TITLE
هل هذا صحيح لصنع لعبة الثعبان

### DISCUSSION THREAD

**Discussion by Kiyotakax** [community/community#108236] [Programming Help]
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Snake Game</title>
	<link rel="stylesheet" href="style.css">
</head>
<body>
	<div id="game-container">
		<canvas id="gc" class="game-canvas" width="400" height="400"></canvas>
	</div>
	<div class="keys">
	  <a class="up arr" onclick="Snake.action('up')">
	    <i class="chevron up"></i></a>
	  <br />
	  <a class="left arr" onclick="Snake.action('left')">
	    <i class="chevron left"></i></a>
	  <a class="down arr" onclick="Snake.action('down')">
	    <i class="chevron down"></i></a>
	  <a class="right arr" onclick="Snake.action('right')">
	    <i class="chevron right"></i></a>
	</div>
	<script type="text/javascript" src="game.js"></script>
</body>
</html>

**Comment by Akash1134** [comment]
Hi @Kiyotakax , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
